components/usage/pkg/apiv1/billing.go

```diff
@@ -326,12 +326,76 @@ func (s *BillingService) ReconcileInvoices(ctx context.Context, in *v1.Reconcile
 		log.WithError(err).Errorf("Failed to udpate usage in stripe.")
 		return nil, status.Errorf(codes.Internal, "Failed to update usage in stripe")
 	}
+	err = s.ReconcileStripeCustomers(ctx)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to reconcile stripe customers.")
+	}
 
 	return &v1.ReconcileInvoicesResponse{}, nil
 }
 
+func (s *BillingService) ReconcileStripeCustomers(ctx context.Context) error {
+	log.Info("Reconciling stripe customers")
+	var costCenters []db.CostCenter
+	result := s.conn.Raw("SELECT * from d_b_cost_center where creationTime in (SELECT max(creationTime) from d_b_cost_center group by id) and nextBillingTime < creationTime and billingStrategy='stripe'").Scan(&costCenters)
```

Comment:
This query is a bit costly (20 sec in prod) but returns the customers that have an outdated billing cycle.
Those are very likely to have missed an invoice event.